src/scenes/home/mentorRequest/mentorRequest.js

```diff
@@ -83,11 +83,15 @@ class MentorRequest extends Component {
     .sort((a, b) => a.name.localeCompare(b.name)) // sort alphabetically
     .map(x => ({ value: x.id, label: x.name }))
 
-  handleOnClick = async (event) => {
+  handleOnClick = (event) => {
     event.preventDefault();
 
     if (!this.isValid()) return;
 
+    this.setState({ isLoading: true }, this.createRequest);
+  }
+
+  createRequest = async () => {
```

Comment:
Should we set a precedent for only async/await going forward or leave it to only promises?

I feel like we should only use one methodology in the repo (new and old)

Reply:
Good question. I prefer using async/await but am fine with Promises too. I actually  just tried it out here out of curiosity to see if it worked (assuming that Webpack / Babel would handle it correctly), since I hadn't seen it used in the OC repo yet.

Main reason I like async/await is simply for less indenting.

I'm good with whatever you want to do tho @kylemh.